src/components/login/Login.tsx

```diff
@@ -0,0 +1,36 @@
+
+import React from 'react'
+
+import './Login.css'
+
+// const Login = ({
+//   handleSubmit,
+//     handleUsernameChange,
+//     handlePasswordChange,
+//     username,
+//     password
+// }) => {
+
+const Login = () => {
+
+  const handleLogin = (event : any) => {
+    event.preventDefault()
+    console.log('login')
```

Comment:
Remove console.log. It's not a good practice.

Reply:
Ok, I useing console log just for testing code. I removed it.